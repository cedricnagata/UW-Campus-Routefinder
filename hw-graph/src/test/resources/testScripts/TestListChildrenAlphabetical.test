# Create a graph
CreateGraph Graph

# Adds three nodes with ascending alphabetical names
AddNode Graph nodeA
AddNode Graph nodeB
AddNode Graph nodeC

# Adds four edges two from A to B and C and two from B to A with different edges
AddEdge Graph nodeA nodeC edgeA
AddEdge Graph nodeA nodeB edgeB
AddEdge Graph nodeB nodeC edgeC
AddEdge Graph nodeB nodeC edgeD

# List the children of the nodes A and B in the graph
ListChildren Graph nodeA
ListChildren Graph nodeB