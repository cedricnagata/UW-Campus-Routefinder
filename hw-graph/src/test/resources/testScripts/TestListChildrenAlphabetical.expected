# Create a graph
created graph Graph

# Adds three nodes with ascending alphabetical names
added node nodeA to Graph
added node nodeB to Graph
added node nodeC to Graph

# Adds four edges two from A to B and C and two from B to A with different edges
added edge edgeA from nodeA to nodeC in Graph
added edge edgeB from nodeA to nodeB in Graph
added edge edgeC from nodeB to nodeC in Graph
added edge edgeD from nodeB to nodeC in Graph

# List the children of the nodes A and B in the graph
the children of nodeA in Graph are: nodeB(edgeB) nodeC(edgeA)
the children of nodeB in Graph are: nodeC(edgeC) nodeC(edgeD)